src/components/NotFound/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { Link } from 'react-router';
+
+export default function NotFound() {
+  return (
+    <>
```

Comment:
```suggestion
    <Fragment>
```
так более читабельно